industrial_extrinsic_cal/CMakeLists.txt

```diff
@@ -82,7 +84,7 @@ generate_messages(
 ## CATKIN_DEPENDS: catkin_packages dependent projects also need
 ## DEPENDS: system dependencies of this project that dependent projects also need
 catkin_package(
-   INCLUDE_DIRS include
+   INCLUDE_DIRS include /usr/include/eigen3
```

Comment:
Please do not use this as it prevents users from using a custom Eigen version.
If you had problems with the Eigen configuration (I see [find_package](https://github.com/ros-industrial/industrial_calibration/pull/56/files#diff-72f0161b5e919a3236e0dabe3a74ea95R24) commented) take a look at https://github.com/ros-industrial/industrial_calibration/pull/60